nodes/transaction-resubmission.md

```diff
@@ -0,0 +1,32 @@
+---
+description: This is a guide on transaction resubmission on Celestia.
+prev:
+  text: "Helpful CLI commands"
+  link: "/nodes/celestia-app-commands"
+---
+
+# Transaction resubmission
+
+In cases where transactions are not included within a 75-second window,
+resubmission is necessary. This is especially important during network
+congestion, as transactions with relatively low fees may not be processed
+even after the network clears up.
+
+Transactions, whether originating from celestia-app or celestia-node,
+won't be regossiped unless there's a new peer, even by nodes with longer TTLs.
```

Comment:
TBH I'm not sure what this really means but I attempted to rephrase

```suggestion
Regardless of whether they originate from celestia-app or celestia-node, transactions will not be re-gossiped, except in the presence of a new peer."
```

Reply:
just that re-gossiping doesn't change between node or app